hackathon_site/dashboard/frontend/src/App.js

```diff
@@ -47,6 +48,7 @@ const UnconnectedApp = () => {
                     <Route exact path="/reports" component={Reports} />
                     <Route exact path="/inventory" component={Inventory} />
                     <Route exact path="/cart" component={Cart} />
+                    <Route exact path="/Incidentform" component={IncidentForm} />
```

Comment:
Minor, I don't really like the inconsistency with the caps in this URL but not the others.

Reply:
should I use kebob or camel case